packages/react-ui-components/src/DateInput/dateInput.js

```diff
@@ -3,7 +3,12 @@ import PropTypes from 'prop-types';
 import enhanceWithClickOutside from 'react-click-outside';
 import moment from 'moment';
 import mergeClassNames from 'classnames';
+import {connect} from 'react-redux';
+import {$transform, $get} from 'plow-js';
 
+@connect($transform({
+    interfaceLanguage: $get('user.preferences.interfaceLanguage')
```

Comment:
can't it have a value like `default` or smth?

Reply:
There is always a default value, because it falls back to the Neos setting `Neos.Neos.userInterface.defaultLanguage` if it's not set in user preferences.

Reply:
(This is set to `en` by default)